src/apps/js_generic/js_generic.cpp

```diff
@@ -13,7 +14,7 @@ namespace ccfapp
   using namespace kv;
   using namespace ccf;
 
-  using Table = kv::Map<std::vector<uint8_t>, std::vector<uint8_t>>;
+  using Table = kv::untyped::Map;
```

Comment:
This is the only instance of this, but its something we probably want: the JS app doesn't need typed wrappers at the C++ level, its tables are bytes-to-bytes. Serialisation can be handled within the JS code.

Reply:
Yes, that's very much the intention!